app/models/miq_expression.rb

```diff
@@ -1719,8 +1719,8 @@ def self.atom_error(field, operator, value)
     value = value.to_s unless value.kind_of?(Array)
 
     dt = case operator.to_s.downcase
-         when "ruby" # TODO
-           :ruby
+         when "ruby"
```

Comment:
I'm just noticing that this is part of an assignment - what do you think about making this a guard clause at the top of this method?